wordpress-mobile/gutenberg-mobile
# Add Enter.key detection to More blocks

This PR adds a simple logic for Enter.key detection in the More block.
The new logic adds a new default empty block after the current - same of the web side logic - regardless of the caret position.

To test:
- Start the demo app
- Put the caret somewhere in the more block
- Tap enter on the soft keyboard, or on the hw keyboard
- It should add a new empty (para) block after the current
